lib/ex_doc/formatter/html/templates/head_template.eex

```diff
@@ -9,6 +9,9 @@
     <%= if config.authors do %>
       <meta name="author" content="<%= Enum.join(config.authors, ", ") %>">
     <% end %>
+    <%= if page.title in ["404", "Search"] do %>
```

Comment:
Instead of checking the title, when you call head_template(...) from these templates, pass `noindex: true`, then you can do:

```suggestion
    <%= if page.noindex do %>
```

:)

Reply:
Changed! Should I add a default `noindex: false` somewhere to prevent `(KeyError) key :noindex not found` exceptions on other pages?

Reply:
I now have `mix test` up and running locally and the answer is a clear "yes". I'll add `noindex: false` to the other places where `head_template/2` is called.

Reply:
Should be fixed now.